src/index.stories.tsx

```diff
@@ -23,207 +23,143 @@ const imageStyle = (src: string) => ({
 	width: "100%",
 });
 
-storiesOf("Slider", module)
-	.add("Default", () => (
-		<div style={{ width: "100vw", height: "100vh" }}>
-			<Slider hasBullets onSlideChange={onSlideChange}>
-				{images.map((image) => (
-					<div key={image} draggable="false" style={imageStyle(image)} />
-				))}
-			</Slider>
-		</div>
-	))
-	.add("Smaller than 100%", () => (
-		<div style={{ width: "500px", height: "500px" }}>
-			<Slider hasBullets>
-				{images.map((image) => (
-					<div key={image} draggable="false" style={imageStyle(image)} />
-				))}
-			</Slider>
-		</div>
-	))
-	.add("With auto sliding", () => (
-		<div style={{ width: "100vw", height: "100vh" }}>
-			<Slider hasBullets auto={2000}>
-				{images.map((image) => (
-					<div key={image} draggable="false" style={imageStyle(image)} />
-				))}
-			</Slider>
-		</div>
-	))
-	.add("With activeIndex (2)", () => (
-		<div style={{ width: "100vw", height: "100vh" }}>
-			<Slider hasBullets activeIndex={2}>
-				{images.map((image) => (
-					<div key={image} draggable="false" style={imageStyle(image)} />
-				))}
-			</Slider>
-		</div>
-	))
-	.add("With activeIndex (interval)", () => {
-		const [activeIndex, setActiveIndex] = useState(2);
-
-		useEffect(() => {
-			const id = setInterval(() => {
-				const index = activeIndex + (1 % images.length);
-				setActiveIndex(index);
-			}, 2000);
-
-			return () => {
-				clearInterval(id);
-			};
-		});
-
-		return (
-			<div style={{ width: "100vw", height: "100vh" }}>
-				<Slider hasBullets activeIndex={activeIndex}>
-					{images.map((image) => (
-						<div key={image} draggable="false" style={imageStyle(image)} />
-					))}
-				</Slider>
-			</div>
-		);
-	})
-	.add("With custom setSlide function", () => {
-		const [activeIndex, setActiveIndex] = useState(0);
-
-		const setSlideCustom = (slide: number) => 1;
-
-		return (
-			<div style={{ width: "100vw", height: "100vh" }}>
-				<Slider
-					hasBullets
-					activeIndex={activeIndex}
-					setSlideCustom={setSlideCustom}
-				>
-					{images.map((image) => (
-						<div key={image} draggable="false" style={imageStyle(image)} />
-					))}
-				</Slider>
-			</div>
-		);
-	})
-	.add("With different `bulletStyle`", () => {
-		return (
-			<div style={{ width: "100vw", height: "100vh" }}>
-				<Slider hasBullets bulletStyle={{ backgroundColor: "#fff" }}>
-					{images.map((image) => (
-						<div key={image} draggable="false" style={imageStyle(image)} />
-					))}
-				</Slider>
-			</div>
-		);
-	})
-	.add("With custom `bullets`", () => {
-		const BulletComponent: BulletComponentType = ({ onClick, isActive }) => (
-			<li
-				style={{
-					width: "25px",
-					height: "25px",
-					backgroundColor: "red",
-					margin: "0 2px",
-					opacity: isActive ? "0.5" : undefined,
-				}}
-				onClick={onClick}
+export default {
+	component: Slider,
+	title: 'Slider',
+	decorators: [(Story: IStory<SliderProps>) => <div style={{ width: "100vw", height: "100vh" }}><Story /></div>],
```

Comment:
To display the sliders also correct in the mobile view and the doc (which are smaler then the viewport). we should use `%` and not `vw`
```suggestion
	decorators: [(Story: IStory<SliderProps>) => <div style={{ width: "100%", height: "100vh" }}><Story /></div>],
```